provider/datagen/src/lib.rs

```diff
@@ -129,9 +202,9 @@ pub(crate) mod rayon_prelude {
 ///
 /// | Mode | Runtime Fallback | Data Size |
 /// |---|---|---|
-/// | [`Runtime`] | Yes, Automatic | Smallest |
-/// | [`RuntimeManual`] | Yes, Manual | Smallest |
-/// | [`Preresolved`] | No | Small |
+/// | [`Runtime`] | Required, Automatic | Smallest |
+/// | [`RuntimeManual`] | Required, Manual | Smallest |
+/// | [`Preresolved`] | Not required | Small |
```

Comment:
The problem with runtime fallback on preresolved is that we don't guarantee that things like `en-001` are present. You really shouldn't be using preresolved if you plan on runtime fallback.

Reply:
do you have a suggestion? I'd rather say required for the first two

Reply:
Now about "N/A" or "Not applicable"

Reply:
Not supported? If it's actually a problem we should discourage people from doing it

Reply:
Ok